src/web/mjs/connectors/VizShonenJump.mjs

```diff
@@ -101,16 +127,75 @@ export default class VizShonenJump extends Connector {
             });
     }
 
+    async _getMangaVolumes(manga) {
+        let auth = await this._getUserInfo();
+        const request = new Request(new URL(manga.id, this.url), this.requestOptions);
+        let data = await this.fetchDOM(request);
+
+        return [...data.querySelectorAll('#o_products tr td:last-of-type a')]
+            .filter(element => {
+                // login required (e.g. age verification) => javascript:void('login to read');
+                if(/(?=javascript:)(?=.*login)/i.test(element.href)) {
+                    return auth.isLoggedIn && auth.isAdult;
```

Comment:
There is some kind of contradiction, how can there be a login link (`javascript:.*login`), when the user is already logged in (`auth.isLoggedIn === true`)?

Reply:
I figured this stuff was off. I spent some time and adjusted the volume paths for login handling that's unique to how they work. Had to revert a couple minor changes with the `fetchDom` calls with queries for the checks, but I think this is generally better.